opentelemetry-api/src/opentelemetry/trace/status.py

```diff
@@ -169,7 +46,7 @@ class Status:
 
     def __init__(
         self,
-        canonical_code: StatusCanonicalCode = StatusCanonicalCode.OK,
+        canonical_code: StatusCode = StatusCode.UNSET,
```

Comment:
should this argument be renamed to status_code? the term "canonical code" seems to be removed from the spec entirely.

This would also include renaming the protected attribute as well.